.github/workflows/ci.yml

```diff
@@ -148,7 +148,7 @@ jobs:
 
       - name: Retrieve PyFluent version
         run: |
-          echo "::set-output name=PYFLUENT_VERSION::$(python -c "from ansys.fluent.core import __version__; print(__version__)")"
+          echo "PYFLUENT_VERSION=$(python -c "from ansys.fluent.core import __version__; print(__version__)")" >> $GITHUB_OUTPUT
```

Comment:
This has been put in place to handle the warning with "set-output" being deprecated module.